projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -149,8 +149,14 @@ build-ami-ubuntu-2004: setup-ami-share deps-ami setup-packer-configs-ami
 release-ami-ubuntu-2004: MAKEFLAGS=
 release-ami-ubuntu-2004: FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami
 release-ami-ubuntu-2004: PACKER_TYPE_VAR_FILES=$(PACKER_AMI_VAR_FILES)
+release-ami-ubuntu-2004: export MANIFEST_OUTPUT=$(FULL_OUTPUT_DIR)/manifest.json
+release-ami-ubuntu-2004: EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
+release-ami-ubuntu-2004: AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)/ami
+release-ami-ubuntu-2004: EXPORT_AMI_DST=$(AMI_S3_DST)/$(GIT_HASH)
+release-ami-ubuntu-2004: LATEST_AMI_S3_URL=$(AMI_S3_DST)/$(LATEST)/ubuntu.raw
```

Comment:
So the Raw images we build using image-builder are produced as a gzipped archive and are uploaded as ubuntu.gz, inside which is the actual image which doesn't have any extension.
```bash


Running post-processor: compress
==> qemu (compress): Using pgzip compression with 96 cores for ./output/ubuntu-2004-kube-v1.22.6.gz
==> qemu (compress): Archiving output/ubuntu-2004-kube-v1.22.6/ubuntu-2004-kube-v1.22.6 with pgzip
==> qemu (compress): Archive ./output/ubuntu-2004-kube-v1.22.6.gz completed

==> Builds finished. The artifacts of successful builds are:
--> qemu: VM files in directory: ./output/ubuntu-2004-kube-v1.22.6
--> qemu: compressed artifacts in: ./output/ubuntu-2004-kube-v1.22.6.gz
```

Are we cool with using `.raw` as the image extension?